eng/scripts/Create-ApiReview.ps1

```diff
@@ -24,6 +24,10 @@ $createReviewScript = (Join-Path $PSScriptRoot .. common scripts Create-APIRevie
 $artifactList = @()
 foreach ($sdk in (Get-AllPackageInfoFromRepo $ServiceDirectory))
 {
+    if ($sdk.ServiceDirectory -ne $ServiceDirectory) {
```

Comment:
I don't have enough context to know if there is a better option but this seems to have a smell to me. Should we be filtering at this level or at the Get-AllPackageInfoFromRepo level, or perhaps some other pipeline filter configuration?

Reply:
This is a bit of a quick fix to unblock identity release. I'm going to be looking at a broader recurse/norecurse option at the higher level because it seems that both are used in different places.